### ISSUE TITLE
Better reporting on Dynamic IPs

### ISSUE THREAD

**Issue by dimitrist74** [ayeowch/bitnodes#17]
Some of us run nodes on Dynamic IPs, making it difficult to spot our nodes or even participate in the incentive program. Some suggestions to solve this:

1. We (nodes) register our Dynamic DNS names (e.g. mynode.now-ip.org) with our service, you periodically resolve all those names to IPs and replace those IPs with names in your reporting.
2. Or, you create a new API call we (nodes) call once a day. In the call we use the name we want to appear in the reporting. When you receive the API call you make a note of the IP it came from plus the name used in call. Then replace those IPs with names in your reporting.

**Comment by ayeowch** [comment]
The Bitnodes Incentive Program does not support dynamic DNS names as they cannot be verified reliably or uniquely.

However, it is possible to allow hostname instead of IP address for the node status page, e.g. https://bitnodes.21.co/nodes/208.118.235.153-8333/. I will keep this ticket open to look into adding this feature.

**Comment by ayeowch** [comment]
@dimitrist74 This is supported now, e.g. https://bitnodes.21.co/nodes/synat.mimuw.edu.pl-8333/.

To activate node with hostname, enter the hostname in https://bitnodes.21.co/#join-the-network and click **CHECK NOW** then click on the link to activate the node. You can then subscribe to alerts by entering an email in the node status page.

**Comment by dimitrist74** [comment]
@ayeowch thanks, I have tested it by registering https://bitnodes.21.co/nodes/node1.weacceptbitcoin.gr-8333/ and it works well.

But in https://bitnodes.21.co/nodes/?q=Greece I see the node (178.128.119.238) listed against the ISP's dynamic name (178.128.119.238.dsl.dyn.forthnet.gr) and not the one I registered (node1.weacceptbitcoin.gr). Is it possible to print the latter instead of the former in the node list?
